omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/RestTestConstants2_2.java

```diff
@@ -16,4 +16,10 @@
 	public final static String PROGRAM_ATTRIBUTE_TYPE_UUID = "d7477c21-bfc3-4922-9591-e89d8b9c8efb";
 	
 	public final static String PATIENT_PROGRAM_UUID = "9119b9f8-af3d-4ad8-9e2e-2317c3de91c6";
+
+	public final static String DIAGNOSIS_UUID = "29bf4fbc-fcdb-4a5b-97ea-0d5c4b4315a1";
+
+	public final static String VOIDED_DIAGNOSIS_UUID = "39bf4fbc-fcdb-4a5b-97ea-0d5c4b4315a7";
+
+	public static final String DIAGNOSIS_TEST_DATA_XML = "patientDiagnosisDataSet.xml";
```

Comment:
This file will not be found. It needs the full path as here https://github.com/openmrs/openmrs-core/pull/2590/commits/d4981cd723b8dd55064c1b58ae6b0bcc4a19665f#diff-0d16b7abc7a5851414804ac8d1b34bb9R31

Reply:
When I add the file as your suggestion above, the file is not found. However, it is found if I just add the filename